### PR TITLE
status req and resp

### PR DESCRIPTION
**Pull request by genghisjahn**
@davidjamesherzog ,  @eytoss , @jinpark , @kamicrafted 

What think ye of this?

### CODE REVIEW

**Comment by genghisjahn** [comment]
It's possible that the `pocket` and `community` arrays will be empty.

File: fixtures/status_request.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+   "game_name":"game 1",
+   "player_name":"hank the frog"
+}
```
Comment (davidjamesherzog) [inline fixtures/status_request.json]: Do we need to pass anything in the body if we are passing the table guid and player guid in the headers?

File: fixtures/status_resp.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+   "game_guid":"abcd1234",
+   "player_guid":"zyxz9876",
```
Comment (davidjamesherzog) [inline fixtures/status_resp.json]: we shouldn't need the guid's returned to us

File: fixtures/status_request.json (hunk 1 above)
Comment (genghisjahn) [inline fixtures/status_request.json]: I screwed this up.  This should be the body for `/join/`, not `/status/`

File: fixtures/status_resp.json (hunk 2 above)
Comment (genghisjahn) [inline fixtures/status_resp.json]: Again, I messed this up.

**Comment by davidjamesherzog** [comment]
LGTM

**Pull request merged**